go/engine/identify2_test.go

```diff
@@ -665,6 +665,33 @@ func TestIdentify2NoSigchain(t *testing.T) {
 	}
 }
 
+// TestIdentifyDeleted tests that identifying a deleted user gives the correct
+// error
+func TestIdentify2Deleted(t *testing.T) {
+	tc := SetupEngineTest(t, "Identify2Deleted")
+	defer tc.Cleanup()
+
+	u := CreateAndSignupFakeUserWithUser(tc, "login")
+	userId := u.User.GetUID()
+	DeleteAccount(tc, u)
+
```

Comment:
Great thx, it's getting stuck on this line:
```
2016-08-24T12:50:32.646766 ▶ [DEBU keybase login_state.go:259] 147 - GetPassphraseStreamWithPassphrase() -> ok
2016-08-24T12:50:32.646918 ▶ [DEBU keybase login_state.go:1015] 148 + Account "DeleteAccount"
```

Is the account name significant?